pkg/cache/cache.go

```diff
@@ -53,14 +59,26 @@ func applyDefaultCacheOptions(opts *Options) *Options {
 		newOpts = *opts
 	}
 	if newOpts.Resync == 0 {
-		newOpts.Resync = 10 * time.Hour
+		newOpts.Resync = getDefaultResyncInterval() * time.Minute
 	}
 	if newOpts.TweakList == nil {
 		newOpts.TweakList = func(*metav1.ListOptions) {}
 	}
 	return &newOpts
 }
 
+func getDefaultResyncInterval() time.Duration {
+	cattleResyncDefaultFromEnv := os.Getenv("CATTLE_RESYNC_DEFAULT")
+	if cattleResyncDefaultFromEnv == "" {
+		return resyncDefault
+	}
+	resyncDefaultFromEnv, err := strconv.Atoi(cattleResyncDefaultFromEnv)
+	if err != nil {
+		return resyncDefault
```

Comment:
Might log error message here that it couldn't read env var, before returning default. Otherwise its a silent error.